mappings/net/minecraft/client/render/Camera.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS ctf net/minecraft/client/render/Camera
+	FIELD a ready Z
+	FIELD b area Lbfi;
+	FIELD c focussedEntity Lahw;
+	FIELD d pos Lcqg;
+	FIELD e blockPos Leu$a;
+	FIELD i pitch F
+	FIELD j yaw F
+	FIELD k thirdPerson Z
+	FIELD l inverseView Z
+	METHOD a updateEyeHeight ()V
+	METHOD a (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+	METHOD a addRotation (FF)V
+		ARG 1 yaw
+		ARG 2 pitch
+	METHOD a update (Lbfi;Lahw;ZZF)V
+		ARG 1 area
+		ARG 2 focussedEntity
+		ARG 3 thirdPerson
+		ARG 4 inverseView
+		ARG 5 partialTicks
+	METHOD a setPos (Lcqg;)V
+		ARG 1 pos
+	METHOD b updateRotation ()V
+	METHOD b moveBy (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+	METHOD b setRotation (FF)V
+		ARG 1 yaw
+		ARG 2 pitch
+	METHOD c getPos ()Lcqg;
+	METHOD c setPos (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+	METHOD d getBlockPos ()Leu;
+	METHOD e getPitch ()F
+	METHOD f getYaw ()F
+	METHOD g getFocussedEntity ()Lahw;
+	METHOD h isReady ()Z
+	METHOD i isThirdPerson ()Z
+	METHOD k getFluidState ()Lcjk;
```

Comment:
This describes the fluid that the camera is encompassed in, right? Maybe it should be described in more detail

Reply:
I just gave this a generic name because I wasn't sure.